README.md

```diff
@@ -15,14 +15,37 @@ Install and Build:
 ``` shell
 sudo make install-cluster
 ``` 
+
+### Virtualenv
+The project can create a Python virtualenv under `env` directory, which avoids collisions with system managed packages. Some of the packages
+used by this project tend to collide with `apt` managed packages (`python3-urllib3` and `python3-psutil`, for example), and will cause some
+system tools to fail. This option properly sets up a Python venv that doesn't install to system Python `dist-packages` path.
+
+This option is by default not applied, since most of the testbeds don't have this problem, because they might not have tools that depend on
+those libraries; but for development environments that do, it can be activated with:
+``` shell
+cd silk
+./bootstrap.sh -v
+``` 
+Install and Build:
+``` shell
+sudo make install-cluster-venv
+```
+
+To run any test script using venv, use one of the following methods:
+
+1. Use full path of Python in venv, i.e. `sudo ./env/bin/python3 silk/unit_tests/silk_run_test.py`.
+2. Activate the venv using `source ./env/bin/activate`, run using `sudo python3 silk/unit_tests/silk_run_test.py`, and `deactive`
```

Comment:
@ctan-g With method2 system python is initiated rather than virtual env as shown below:

pi@ubuntu:~/silk$ source ./env/bin/activate
(env) pi@ubuntu:~/silk$ sudo which python
/usr/bin/python
(env) pi@ubuntu:~/silk$ sudo which python3
/usr/bin/python3
(env) pi@ubuntu:~/silk$ sudo which ./env/bin/python3
./env/bin/python3
(env) pi@ubuntu:~/silk$ sudo python3 silk/unit_tests/silk_run_test.py
Traceback (most recent call last):
  File "silk/unit_tests/silk_run_test.py", line 18, in <module>
    from silk.tests import silk_run
  File "/usr/local/lib/python3.6/dist-packages/silk-1.0.0-py3.6.egg/silk/tests/silk_run.py", line 30, in <module>
    import silk.tests.testcase
  File "/usr/local/lib/python3.6/dist-packages/silk-1.0.0-py3.6.egg/silk/tests/testcase.py", line 38, in <module>
    from silk.tools.otns_manager import OtnsManager
  File "/usr/local/lib/python3.6/dist-packages/silk-1.0.0-py3.6.egg/silk/tools/otns_manager.py", line 31, in <module>
    import pandas
ModuleNotFoundError: No module named 'pandas'
(env) pi@ubuntu:~/silk$ deactivate
pi@ubuntu:~/silk$

Reply:
What do the crosses out lines mean? Is it working or not working?

Reply:
Maybe I'll just remove method 2 if it's not working

Reply:
method 2 is not working. 
Nothing is crossed out. For cli output the github formatting was showing crossed out lines. Edited it.

Reply:
Thanks for testing and clarifying! I'll remove method 2 since it's not working as I expected.